src/pages/Index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { Button } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { logIn } from '../state/user';
+import './Index.less';
+
+class Index extends Component {
+  render() {
+    const { logIn: onClick } = this.props;
+    return (
+      <div className="index-page">
+        <h1>Welcome! We&#39;re glad you make it this far.</h1>
```

Comment:
Should be "you made it" / "you've made it"?

Reply:
yes. I just copied it from wireframes. Fixed.